vllm_ascend/platform.py

```diff
@@ -103,17 +107,35 @@ def mem_get_info(cls) -> Tuple[int, int]:
 
     @classmethod
     def check_and_update_config(cls, vllm_config: VllmConfig) -> None:
+        compilation_config = vllm_config.compilation_config
+        if compilation_config.level != CompilationLevel.NO_COMPILATION:
+            logger.info("[NPU] Forcing NO_COMPILATION compilation level")
```

Comment:
print current compilation_config.level to log as well. and change to warning level.